lib/push.js

```diff
@@ -0,0 +1,186 @@
+var client = require('./rest'),
+    constant = require('./constants'),
+    logger = require('./logger'),
+    dispatcher = require('./dispatcher'),
+    channels = require('./push/channels');
+
+function timezone() {
```

Comment:
Is it possible to use the moment library for this function ? http://momentjs.com/timezone/